source/mir/appender.d

```diff
@@ -26,7 +26,7 @@ struct ScopedBuffer(T, size_t bytes = 4096)
 
     private enum size_t _bufferLength =  bytes / T.sizeof + (bytes % T.sizeof != 0);
     private T[] _buffer;
-    size_t _currentLength;
+    private size_t _currentLength;
```

Comment:
It has to be public because it is used in some very low level weird code in mir-ion

Reply:
Ok. Resolved via b486406.